### DISCUSSION TITLE
Assistant APi Streaming with GPT 3.5 and 4o

### DISCUSSION THREAD

**Discussion by sreekanthedayar** [openai/openai-dotnet#29] [Q&A] [answered]
We had our own implementation in .net which works well without streaming. Does this library support streaming for Assistant API with GPT3.5 and 4o?

**Comment by trrwilson** [comment, marked as answer]
Absolutely! You can see a walkthrough of the usage for Assistants streaming in the readme:

https://github.com/openai/openai-dotnet?tab=readme-ov-file#how-to-use-streaming-and-gpt-4o-vision-with-assistants
